currency_rate_update/models/res_currency_rate_provider_ECB.py

```diff
@@ -16,6 +16,11 @@ class ResCurrencyRateProviderECB(models.Model):
 
     service = fields.Selection(selection_add=[("ECB", "European Central Bank")])
 
+    def _get_close_time(self):
```

Comment:
Shouldn't this go to base class as well?

Reply:
Exactly, you're right, sorry. It's already changed.